### PR TITLE
 📖 [Story videos] Add video documentation for Google cache

### PR DESCRIPTION
**Pull request by mszylkowski**
Document the `cache="google"` attribute.

### CODE REVIEW

**Comment by gmajoulet** [comment]
Should we link to it from the Story documentation? I don't think many people go read the amp-video doc when using a Story

**Comment by mszylkowski** [comment]
I was planning to add it to the guides in:
- https://amp.dev/documentation/guides-and-tutorials/start/create_successful_stories/?format=stories#video
- https://amp.dev/documentation/guides-and-tutorials/optimize-and-measure/optimize_amp/?format=stories

but those are added in the amp.dev repo. Do you have any other suggestions on where to add it?

**Comment by gmajoulet** [comment]
that sounds good! Wdyt of an additional mention in the amp-story doc?

File: extensions/amp-story/amp-story.md [hunk 1]
```diff
@@ -395,6 +395,20 @@ This same image can be used for both mobile portrait and landscape desktop using
 <amp-img src="cat.jpg" alt="..." object-position="75% 40%"></amp-img>
 ```
 
+##### Optimize `amp-video` by using a video cache on origin documents
```
Comment (gmajoulet) [inline extensions/amp-story/amp-story.md:398]:
```suggestion
##### Optimize `amp-video` by using a free Google hosted video cache on origin documents
```

File: extensions/amp-story/amp-story.md [hunk 2]
```diff
@@ -395,6 +395,20 @@ This same image can be used for both mobile portrait and landscape desktop using
...
+The `<amp-video>` element on stories supports specifying a video cache to be used on origin documents through the attribute `cache`.
+The video cache can serve copies of the video at multiple bitrates, reducing the server's workload on delivering videos.
+The appropriate video quality plays on the story, according to the network conditions of the viewer.
+
+Example:
+
+```html
+<amp-video layout="fill" poster="img.png" cache="google">
+  <source src="video.mp4" type="video/mp4">
+</amp-video>
```
Comment (gmajoulet) [inline extensions/amp-story/amp-story.md:409]: Please copy the relevant parts of the existing external communication that has already been reviewed, so it's all consistent

File: extensions/amp-video/0.1/amp-video.md [hunk 3]
```diff
@@ -156,6 +156,13 @@ Indicates the album/collection the video was taken from, specified as a string.
 
 Indicates the name/title of the video, specified as a string. If not provided, the Media Session API helper uses either the `aria-label` attribute or falls back to the page's title.
 
+[filter formats="stories"]
+
+#### cache
```
Comment (gmajoulet) [inline extensions/amp-video/0.1/amp-video.md:161]: Unless we document how to implement your own system, we should keep this specific to the Google provided video caching system

File: extensions/amp-video/0.1/amp-video.md [hunk 4]
```diff
@@ -156,6 +156,13 @@ Indicates the album/collection the video was taken from, specified as a string.
...
+
+Indicates the Google video cache should store and serve the video by adding `cache="google"`. The video cache will fetch and store the video contents periodically, reducing serving costs for videos, and generating transcodes with different quality settings that adapt the bitrate to the network conditions.
```
Comment (gmajoulet) [inline extensions/amp-video/0.1/amp-video.md:163]: Should we use this snippet even within `amp-story.md`? I got a bit lost with the origin/cache/video cache wording, this one looks better

Comment (mszylkowski) [reply extensions/amp-video/0.1/amp-video.md:163]: We need to keep the first sentence on `amp-story.md` so we exaplain it's on `amp-video` but I can make the rest the same as here

**Review by gmajoulet — APPROVED** [review]

**Pull request merged**